apps/backend/src/deposits/pos-deposit.service.ts

```diff
@@ -78,29 +59,25 @@ export class PosDepositService {
       .getRawMany();
   }
 
-  async findAllByLocationAndDate(event: ReconciliationEvent) {
-    return await this.queryPOS(
-      event,
-      await this.getMerchantIdsByLocationId(event)
-    );
-  }
-
   async createPOSDepositEntity(
     data: POSDepositEntity
   ): Promise<POSDepositEntity> {
     return await this.posDepositRepo.save(this.posDepositRepo.create(data));
   }
 
-  async updatePOSDepositEntity(
-    deposit: Partial<POSDepositEntity>,
```

Comment:
There should ideally be no use for `Partial types` in our work so far, all our entities are fully defined and loaded. 

This issue stems because sql queries are used in getting the data and creating the entity classes. 

This is unsafe because it will overwrite the data...if we save the entity directly (which we arent doing here)